src/routers/expenses.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const expensesRouter = express.Router();
+const expenceController = require('../controllers/expenses.js');
+
+expensesRouter.get('/', expenceController.getAll);
+
+expensesRouter.get('/:expenseId', expenceController.getOne);
+
+expensesRouter.post('/', expenceController.add);
+
+expensesRouter.patch('/:expenseId', expenceController.updateExpense);
```

Comment:
The typographical error is found where 'expenceController' is used. Please correct it to 'expenseController'. Fix all typos